app/models/registration_wizard.rb

```diff
@@ -307,6 +307,7 @@ def steps
       your_role
       your_employer
       school_not_in_england
+      childcare_provider_not_in_england
```

Comment:
This page is pretty much inaccessible in actuality, since in_england? is hard coded to true for ChildcareProviders. However, better to have the page actually working rather than not.